app/ts/ClaimFundsUI/ClaimFunds.tsx

```diff
@@ -261,37 +292,31 @@ export const ClaimFunds = ({ isAugurExtraUtilitiesDeployedSignal, updateTokenBal
 			return await redeemStake(writeClient, reportingParticipants, disputeWindows)
 		}
 
-		const refreshClaim = async () => {
-			updateTokenBalancesSignal.value++
-			selectedDisputes.value = []
-			selectedReports.value = []
-			return await queryForData(writeClient).catch(showUnexpectedError)
-		}
 		const claimWinningShares = async () => {
 			return await claimMarketWinnings(writeClient, selectedShares.value)
 		}
-		const refreshShares = async () => {
-			updateTokenBalancesSignal.value++
-			selectedShares.value = []
-			return await queryForData(writeClient).catch(showUnexpectedError)
-		}
+
 		const claimForkDisputes = async () => {
 			if (isAugurExtraUtilitiesDeployedSignal.deepValue !== true) throw new Error('extra utils not deployed')
 			const selected = Array.from(selectedForkedCrowdSourcers.value) // Winning Initial Reporter or Dispute Crowdsourcer bonds the msg sender has stake in
 			if (selected.length === 0) throw new Error('nothing to claim')
 			return await forkReportingParticipants(writeClient, selected)
 		}
-		const refreshClaimForkDisputes = async () => {
-			updateTokenBalancesSignal.value++
-			selectedForkedCrowdSourcers.value = []
-			return await queryForData(writeClient).catch(showUnexpectedError)
-		}
+
+		const isForkDisputesDisabled = useComputed(() => isAugurExtraUtilitiesDeployedSignal.deepValue !== true)
 
 		return <div class = 'subApplication'>
 			<section class = 'subApplication-card'>
 				<div style = 'display: grid; width: 100%; gap: 10px;'>
 					<div style = 'display: grid; width: 100%; gap: 10px;'>
 						<DisplayShareData loading = { loading } pathSignal = { pathSignal } availableShareData = { availableShareData } selectedShares = { selectedShares }/>
+						<LoadingButton
+							isLoading = { isLoadingShareData }
+							startLoading = { queryShareData }
+							disabled = { useComputed(() => false) }
```

Comment:
Would this be equivalent, and potentially more clear?
```suggestion
							disabled = { useSignal(false) }
```